test-requirements.txt

```diff
@@ -1,4 +1,3 @@
-aiohttp >= 3.6.2
```

Comment:
aiohttp was removed?

Reply:
idk why that happened, it also removed it from `requirements.txt` but i added that one back in manually, should i add back here too?

Reply:
Well it looks like tests passed anyway, and all the tests are stubs anyway right now. We can add this back when the tests are fixed.